client/kate/Cargo.toml

```diff
@@ -7,30 +7,38 @@ edition = "2018"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-dusk-plonk = { path = "../plonk", default-features = false, optional = true }
+dusk-plonk = { version = "0.8.2", default-features = false, optional = true }
```

Comment:
Earlier we used local plonk code, now using dusk-plonk code. Same question as above - when are we using which?

Reply:
We are using our overridden version, see [above comment](https://github.com/maticnetwork/substrate/pull/6#discussion_r755191982).
You can also verify that the resolved version is right in [Cargo.lock:1502](https://github.com/maticnetwork/substrate/blob/4484f6c13dfe25a9eea79b1c29638dfcdcf0a037/Cargo.lock#L1502)